citro3d/examples/triangle.rs

```diff
@@ -87,13 +93,22 @@ fn main() {
         }
 
         instance.render_frame_with(|instance| {
+            instance
```

Comment:
BTW my IDE suggests these `render_frame_with` calls are duplicate code and should be consolidated into a function.